benchmark/C/Savina/bndbuffer/ProdConsBenchmark.lf

```diff
@@ -0,0 +1,393 @@
+/*
+ * Concurrency benchmark from the Savina benchmark suite.
+ * See https://shamsimam.github.io/papers/2014-agere-savina.pdf.
+ * @author Hannes Klein
+ * @author Marten Lohstroh
+ * @author Soroush Bateni
+ */
+ 
+target C {
+    flags: "-lm",
+    logging: DEBUG,
+    keepalive: true,
+    timeout: 2 secs
+};
+
+import BenchmarkRunner from "../BenchmarkRunner.lf";
```

Comment:
Some benchmarks rely on the benchmark runner and some don't. Does the latest version of `BenchmarkRunner` have an adverse effect on performance? If not, I think we can use `BenchmarkRunner` consistently in all the benchmarks.

Reply:
We're still in the process of evaluating whether there is any merit to have a benchmark runner in the C tests. We'll remove whatever we don't use later on.